tools/filters/gff/gtf_filter_by_attribute_values_list.xml

```diff
@@ -1,8 +1,8 @@
-<tool id="gtf_filter_by_attribute_values_list" name="Filter GTF data by attribute values_list" version="0.1">
+<tool id="gtf_filter_by_attribute_values_list" name="Filter GTF data by attribute values_list" version="0.2">
     <description></description>
-    <command interpreter="python">
-      gtf_filter_by_attribute_values_list.py $input $attribute_name $ids $output
-    </command>
+    <command><![CDATA[
+python '$__tool_directory__/gtf_filter_by_attribute_values_list.py' '$input' $attribute_name '$ids' '$output'
```

Comment:
Since we are upgrading the tool version now, it makes sense to also add a `<requirement>` for python, as originally suggested by @mvdbeek in the original PR.